tests/fixtures/issues/issue-1413.php

```diff
@@ -51,7 +51,7 @@
             'RenderingEngine_Version' => 'unknown',
             'RenderingEngine_Maker' => 'Google Inc',
         ],
-        'lite' => true,
+        'lite' => false,
```

Comment:
I mentioned this in #1528, but all of these "lite" changes are because of the change to "general Tablet" as the default device for these patterns unless they have the "Mobile" keyword in the UA.

These particular tests have a device that is defined as a Phone for the Full file, so has "Mobile Phone" as the device type, but the Lite file doesn't have that device, so it still thinks these should be tablet.  There's not really a way around this, and from my analysis, this should actually happen a lot less (80-90% vs. 10-20%) than the other way around (tablets being detected as phones).